### PR TITLE
Fix deployment file name in the tutorial

### PR DESCRIPTION
**Pull request by Hongbo-Miao**
Just a small fix. 😃
After running previous step `prefect deployment build ./basic_flow.py:basic_flow -n test-deployment -q test`, the file got generated is `basic_flow-deployment.yaml` instead of `deployment.yaml`.

BTW, I love Prefect, so neat!

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

File: docs/ui/cloud-getting-started.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ What did we do here? Let's break down the command:
 - `-n test-deployment` is an option to specify a name for the deployment.
 - `-q test` specifies a work queue for the deployment. The deployment's runs will be sent to any agents monitoring this work queue.
 
-The command outputs two files: `basic_flow-manifest.json` contains workflow-specific information such as the code location, the name of the entrypoint flow, and flow parameters. `deployment.yaml` contains details about the deployment for this flow.
+The command outputs two files: `basic_flow-manifest.json` contains workflow-specific information such as the code location, the name of the entrypoint flow, and flow parameters. `basic_flow-deployment.yaml` contains details about the deployment for this flow.
```
Comment (tpdorsey) [inline docs/ui/cloud-getting-started.md:215]: In addition, we don't create the manifest anymore, so perhaps let's update this to something like "The command outputs `basic_flow-deployment.yaml`, which contains"?

**Review by tpdorsey — COMMENTED** [review]
Thank you for catching this. Great contribution. Minor additional comment on this.

File: docs/ui/cloud-getting-started.md (hunk 1 above)
Comment (Hongbo-Miao) [reply docs/ui/cloud-getting-started.md]: Done, thanks!

**Review by tpdorsey — APPROVED** [review]
Ty this looks great

**Pull request merged**